src/app/tests/suites/certification/Test_TC_BINFO_2_3_Simulated.yaml

```diff
@@ -23,9 +23,22 @@ config:
     endpoint: 0
 
 tests:
+    #- label: "Wait for the device to be commissioned"
+    #  cluster: "DelayCommands"
+    #  command: "WaitForCommissioning"
+
     - label: "Wait for the device to be commissioned"
-      cluster: "DelayCommands"
-      command: "WaitForCommissioning"
+      PICS: PICS_SKIP_SAMPLE_APP
```

Comment:
@manjunath-grl same for all simulated tests , need to remove the entire step which used to be the wait for device to be commissioned , along with the user prompt

Reply:
Removed the prompt step.